Resources/views/PlatformUI/shell.html.twig

```diff
@@ -45,7 +45,17 @@
         <p>{{ 'javascript.disabled.required'|trans }}</p>
         <p><strong>{{ 'javascript.enable'|trans }}</strong></p>
     </noscript>
+    {% javascripts
+        'bundles/bazingajstranslation/js/translator.min.js'
+        'js/translations/config.js'
+    %}
+        <script src="{{ asset_url }}"></script>
+    {% endjavascripts %}
+    {# We only load the translations for the eZPlatformUIBundle JS files #}
+    <script src="{{ url('bazinga_jstranslation_js', { 'domain': 'editorial' }) }}"></script>
```

Comment:
This loads all translations afaik, but depending on how we let users select UI languages in the future we should probably limit it to not load all. see https://github.com/willdurand/BazingaJsTranslationBundle/blob/master/Resources/doc/index.md#locales